src/seedu/addressbook/data/AddressBook.java

```diff
@@ -61,6 +61,13 @@ public void clear() {
         allPersons.clear();
     }
 
+    /**
+     * Sort all persons in the address book by their names lexicographically.
```

Comment:
Good that you have added comments. But Always use plural form for method header comment or class header comment to adhere to the coding standards. E.g. Sorts all person